src/main/java/com/trandiepphuong/models/Task.java

```diff
@@ -0,0 +1,20 @@
+package com.trandiepphuong.models;
+
+import lombok.Data;
+
+import javax.persistence.*;
+
+@Data
+@Entity
+public class Task {
+
+    @Id
+    @GeneratedValue(strategy= GenerationType.IDENTITY)
+    private int id;
+
+    @Column(name = "content")
```

Comment:
Không cần thiết phải thêm 
```suggestion
    @Column
```

Reply:
Done